packages/turf-meta/index.js

```diff
@@ -822,6 +739,29 @@ function feature(geometry, properties) {
     };
 }
 
+/**
+ * Create LineString
+ * @turf/helpers
+ *
+ * @private
+ * @param {Array<Array<number>>} coordinates Line Coordinates
+ * @param {Object} properties Properties
+ * @returns {Feature<LineString>} GeoJSON LineString Feature
+ */
+function lineString(coordinates, properties) {
```

Comment:
@DenisCarriere I don't get why exactly we need/want to have no dependencies in `meta`, if we would actually need `helpers` for instance. Is this faster maybe?